packages/core/frontend/sirius-components-omnibox/src/Omnibox.tsx

```diff
@@ -70,10 +72,13 @@ export const Omnibox = ({ open, initialContextEntries, onClose }: OmniboxProps)
   const [state, setState] = useState<OmniboxState>({
     contextEntries: initialContextEntries,
     queryHasChanged: true,
+    customDialogId: null,
   });
 
-  const { setSelection } = useSelection();
   const { getOmniboxCommands, loading, data } = useOmniboxCommands();
+  const { executeOmniboxCommand } = useExecuteOmniboxCommand();
+
+  const omniboxCommandDialogExtensions = useComponents(omniboxCommandDialogExtensionPoint);
```

Comment:
Use a Data extension point here with a `canHandle`, this way we don't have to have a strict contract on the frontend/backend ids.

Reply:
Done